lua/modules/menu.lua

```diff
@@ -1516,29 +1516,37 @@ if DEV_MODE == true then
 							onDone = function()
 								aiInput:focus()
 							end,
-						}).Y = targetInputPosition[2]
+						}).Y =
+							targetInputPosition[2]
 						ease:outBack(aiCharacter.pos, 0.22, {
 							onDone = function() end,
-						}).Y = targetCharacterPosition[2]
+						}).Y =
+							targetCharacterPosition[2]
 						ease:outBack(aiCharacterBubble.pos, 0.24, {
 							onDone = function() end,
-						}).Y = targetBubblePosition[2]
+						}).Y =
+							targetBubblePosition[2]
 						ease:outBack(modelCombo.pos, 0.24, {
 							onDone = function() end,
-						}).Y = targetModelComboPosition[2]
+						}).Y =
+							targetModelComboPosition[2]
 					else
 						ease:inBack(aiInput.pos, 0.2, {
 							onDone = function() end,
-						}).Y = targetInputPosition[2]
+						}).Y =
+							targetInputPosition[2]
 						ease:inBack(aiCharacter.pos, 0.22, {
 							onDone = function() end,
-						}).Y = targetCharacterPosition[2]
+						}).Y =
+							targetCharacterPosition[2]
 						ease:inBack(aiCharacterBubble.pos, 0.24, {
 							onDone = function() end,
-						}).Y = targetBubblePosition[2]
+						}).Y =
+							targetBubblePosition[2]
 						ease:inBack(modelCombo.pos, 0.24, {
 							onDone = function() end,
-						}).Y = targetModelComboPosition[2]
+						}).Y =
+							targetModelComboPosition[2]
```

Comment:
There may be an issue with your Lua editor setup @gdevillele?
It keeps inserting those line breaks.

Reply:
I don't understand where this is coming from :( 
I'd like to compare our setups.

I have those 3 extensions
/Users/gaetan/screenshots/Screenshot 2025-05-15 at 09.37.33.png

Reply:
Ok. It looks like Stylua is in cause.

Reply:
my `.stylua.toml`

```toml
syntax = "Luau"
column_width = 120
line_endings = "Unix"
indent_type = "Tabs"
indent_width = 4
quote_style = "AutoPreferDouble"
call_parentheses = "Always"
collapse_simple_statement = "Never"
space_after_function_names = "Never"

[sort_requires]
enabled = false
```